src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -48,15 +48,15 @@ const AccordionPanel = forwardRef(
           aria-expanded={active}
           onClick={onPanelChange}
           onMouseOver={event => {
-            setHover(theme.dark ? 'light-4' : 'dark-3');
+            setHover(theme.accordion.hover && theme.accordion.hover.color);
```

Comment:
This will set `hover` to `false` if there is no value in the theme. I think we we should set it to undefined instead:
`setHover((theme.accordion.hover && theme.accordion.hover.color) || undefined);`